Sprint-1/interpret/time-format.js

```diff
@@ -1,24 +1,30 @@
-const movieLength = 8784; // length of movie in seconds
+const movieLength = 999999; // length of movie in seconds
 
 const remainingSeconds = movieLength % 60;
 const totalMinutes = (movieLength - remainingSeconds) / 60;
+console.log("the total" + totalMinutes);
 
 const remainingMinutes = totalMinutes % 60;
 const totalHours = (totalMinutes - remainingMinutes) / 60;
 
 const result = `${totalHours}:${remainingMinutes}:${remainingSeconds}`;
 console.log(result);
 
-// For the piece of code above, read the code and then answer the following questions
+// For the piece of code above, read the ode and then answer the following questions
 
 // a) How many variable declarations are there in this program?
+//  6 variabls
 
 // b) How many function calls are there?
-
+//    1
 // c) Using documentation, explain what the expression movieLength % 60 represents
+//   The remainder (%) operator returns the remainder left over when one operand is divided by a second operand. It always takes the sign of the dividend.fb
 
 // d) Interpret line 4, what does the expression assigned to totalMinutes mean?
+//  calculate the total minutes by subtracting remainingSeconds from movieLength and then dividing by 60 to convert the remaining seconds into minutes
 
 // e) What do you think the variable result represents? Can you think of a better name for this variable?
+// its using tamplat to store variable and any tex /the better name could be time
```

Comment:
`time` may also imply value like "8:45 pm" or "20:45". Can you come out with a more suitable name?

Reply:
i think.   movieRuntime